community/py-magic/APKBUILD

```diff
@@ -0,0 +1,50 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-magic
+_pkgname=${pkgname#py-*}
+_pkgreal=${pkgname/py/python}
+pkgver=0.4.12
+pkgrel=0
+pkgdesc="A python wrapper for libmagic"
+url="https://pypi.python.org/pypi/python-magic/"
+arch="noarch"
+license="MIT"
+depends="python libmagic"
```

Comment:
`python` is old name of the package `python2`, they are de facto synonyms, so this would make py3-magic depend on python2 that doesn’t make sense. Please look at https://wiki.alpinelinux.org/wiki/APKBUILD_examples:Python.